### ISSUE TITLE
`@orderby` should throw an error if incorrect column syntax is used

### ISSUE THREAD

**Issue by metanoid** [queryverse/Query.jl#288]
Using `Query.jl` in conjunction with `DataFrames.jl` I find that I often make silly mistakes with the syntax, such as:

```julia
df |>
      @orderby(:IssueDate) |>
      DataFrame
```

instead of 
```julia
df |>
      @orderby(_.IssueDate) |>
      DataFrame
```

In such cases, `Query.jl` simply ignores the `@orderby` line, and does so silently.

I think that if the arguments to `@orderby` do not resolve to something in `df` that can be ordered, there should be a warning thrown, if not an error.

**Comment by davidanthoff** [comment]
Agreed! Or, even better, it could just work with `@orderby(:IssueDate)` as well :)
